rockcraft.yaml

```diff
@@ -133,7 +133,7 @@ parts:
       - python3-pip
     overlay-script: |
       mkdir -p $CRAFT_PART_INSTALL/opt/spark8t/python/dist
-      pip install --target=${CRAFT_PART_INSTALL}/opt/spark8t/python/dist  https://github.com/canonical/spark-k8s-toolkit-py/releases/download/v0.0.2/spark8t-0.0.2-py3-none-any.whl
+      pip install --target=${CRAFT_PART_INSTALL}/opt/spark8t/python/dist  git+https://github.com/canonical/spark-k8s-toolkit-py.git@dpe-3398-fix-service-account-permission
```

Comment:
This should be fixed once we do a new `spark8t` release, maybe before this PR gets merged?

Reply:
`spark8t` `v0.0.3` has been released. We can now get this changed.

Reply:
Done!